plugins/kotlin/idea/tests/testData/formatter/trailingComma/whenEntry/WhenEntry.after.kt

```diff
@@ -69,8 +66,7 @@ fun foo(x: Any) {
 
     when (val c = x) {
         1, 2,
-        3, /**/
-        -> {
+        3 /**/ -> {
```

Comment:
The same as above, the trailing comma is missing

Reply:
Droped this change